diff --git a/CMakeLists.txt b/CMakeLists.txt
index ff7782d..c6b40a8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,10 +45,10 @@ target_link_libraries(qpdftools
 
 if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
 
-  install(TARGETS qpdftools DESTINATION /usr/bin)
-  install(DIRECTORY resources/hicolor DESTINATION /usr/share/icons)
-  install(FILES ${QM} DESTINATION /usr/lib/qpdftools)
-  install(FILES resources/io.github.silash35.qpdftools.desktop DESTINATION /usr/share/applications)
-  install(FILES resources/io.github.silash35.qpdftools.appdata.xml DESTINATION /usr/share/metainfo)
+  install(TARGETS qpdftools DESTINATION /app/bin)
+  install(DIRECTORY resources/hicolor DESTINATION /app/share/icons)
+  install(FILES ${QM} DESTINATION /app/lib/qpdftools)
+  install(FILES resources/io.github.silash35.qpdftools.desktop DESTINATION /app/share/applications)
+  install(FILES resources/io.github.silash35.qpdftools.appdata.xml DESTINATION /app/share/metainfo)
 
 endif (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
